pkg/lib/config/feature_web3.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+var _ = FeatureConfigSchema.Add("Web3FeatureConfig", `
+{
+	"type": "object",
+	"additionalProperties": false,
+	"properties": {
+		"nft": { "$ref": "#/$defs/Web3NFTFeatureConfig" }
+	}
+}	
+`)
+
+type Web3FeatureConfig struct {
+	NFT *Web3NFTFeatureConfig `json:"nft,omitempty"`
+}
+
+var _ = FeatureConfigSchema.Add("Web3NFTFeatureConfig", `
+{
+	"type": "object",
+	"additionalProperties": false,
+	"properties": {
+		"maximum": {"type": "integer"}
```

Comment:
When saving the config, we should respect this settings. See `validateBasedOnFeatureConfig` for details.